<launch>
    <node pkg="hector_scanmatcher" type="hector_scanmatcher" name="hector_scanmatcher" output="screen">
        <param name="base_frame" value="base_stabilized" />
        <param name="odom_frame" value="nav"/>
        <param name="map_resolution" value="0.025"/>
        <param name="output_timing" value="true"/>
        <param name="use_tf" value="true"/>
        <param name="scan_topic" value="scan"/>
        <param name="map_size" value="2048"/>
<!--        <param name="pub_drawings" value="true"/>
        <param name="pub_debug_output" value="true"/>-->
        <param name="pub_map_odom_transform" value="false"/>
    </node>
</launch>
